.circleci/cimodel/data/pytorch_build_definitions.py

```diff
@@ -316,6 +317,9 @@ def instantiate_configs():
             parms_list_ignored_for_docker_image.append("coverage")
             python_version = fc.find_prop("pyver")
 
+        if is_noarch:
+            parms_list_ignored_for_docker_image.append("noarch")
```

Comment:
```suggestion
           restrict_phases = ["build"] if restrict_phases is None else restrict_phases
           restrict_phases.extend(["test", "noarch_test"])
```
I believe this code would not change the name of the existing build but instead would create a separate CI test job that only runs noarch tests, which would help with isolating the tests, if that is to be desired

Reply:
I think we don't want this change? I don't want to split out the noarch tests into one config that only runs them, because I don't have any evidence that sharding them separately would be a benefit (noarch is much faster than normal tests so it would be super lopsided).

Reply:
Yea, that's fine in that case.